packages/venia-concept/webpack.config.js

```diff
@@ -77,7 +79,11 @@ module.exports = async function(env) {
                     use: [
                         {
                             loader: 'babel-loader',
-                            options: { ...babelOptions, cacheDirectory: true }
+                            options: {
+                                cacheDirectory: false,
```

Comment:
Did we try a cache directory and it was buggy?

Reply:
No, this was supposed to be temporary while I was upgrading, to make sure it didn't interfere. I'll set this back to `true` now.

Reply:
Done.